image/meta-gcp-iot/recipes-gcp/python-docs-samples/files/start-mqtt-example.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+#
+
+export PROJECT_ID="$(echo $FULL_PROJECT | cut -f2 -d ':')"
+export REGION_ID='us-central1'
+
+export REGISTRY_ID='mender-demo'
+export DEVICE_ID='mender_ota_rasp3'
+
+python cloudiot_mqtt_example.py \
+       --project_id "${PROJECT_ID}" \
+       --registry_id "${REGISTRY_ID}" \
+       --device_id "${DEVICE_ID}" \
+       --algorithm RS256 \
```

Comment:
@Kcr19 should this parameter be RSA256 or is RS256 correct?

Reply:
RS256 should work

Reply:
with the activation agent - I wasn't initially sure if we still wanted the sample. But I guess it is nice to have a step that the user sees data can be published.

We only allow one MQTT connection per device - so we should probably modify the activation agent script to also check for /data/mender/pre-auth true and quit

Reply:
@ptone I can remove the sample script if there is no value in it; I wasn't sure if your activation agent was intended to replace it.  Let me know your preference and I'll be sure to update this PR before merging.